src/main/java/dev/jbang/Script.java

```diff
@@ -330,6 +356,10 @@ public void createJarFile(File path, File output) throws IOException {
 		if (mainclass != null) {
 			manifest.getMainAttributes().put(Attributes.Name.MAIN_CLASS, mainclass);
 		}
+		if (buildJdk > 0) {
+			String val = buildJdk >= 9 ? Integer.toString(buildJdk) : "1." + buildJdk;
```

Comment:
why do we care about 1.x here now ? shouldn't it still be 8 instead 1.8 ?

Reply:
AFAIK Java 8 is still numbered 1.8 when you ask for it's version, right?

Reply:
(This is an "official" field used by Maven for example, so I'm following what it does)

Reply:
Conversation moved to Issue #183